fink_science/random_forest_snia/processor.py

```diff
@@ -337,85 +332,68 @@ def rfscore_sigmoid_elasticc(
 
     # Perform the fit + classification (default model)
     >>> args = [F.col(i) for i in what_prefix]
-    >>> args += [F.col('cdsxmatch'), F.col('diaSource.nobs')]
-    >>> args += [F.col('diaObject.ra'), F.col('diaObject.decl')]
-    >>> args += [F.col('diaObject.hostgal_ra'), F.col('diaObject.hostgal_dec')]
+    >>> args += [ F.col('diaObject.hostgal_dec'), F.col('diaObject.hostgal_ra')]
+    >>> args += [F.col('diaObject.hostgal_snsep')]
     >>> args += [F.col('diaObject.hostgal_zphot')]
-    >>> args += [F.col('diaObject.hostgal_zphot_err'), F.col('diaObject.mwebv')]
+    >>> args += [F.col('diaObject.hostgal_zphot_err')]
+    >>> args += [F.col('diaObject.ra'), F.col('diaObject.decl')]
     >>> df = df.withColumn('pIa', rfscore_sigmoid_elasticc(*args))
```

Comment:
The order of columns is not correct. `diaObject.ra` and `diaObject.decl` should be just before `diaObject.hostgal_dec` (following the definition of `rfscore_sigmoid_elasticc`).

Reply:
ok fixed